conf/distro/rpb-wayland.conf

```diff
@@ -4,3 +4,7 @@ DISTRO_NAME = "Reference-Platform-Build-Wayland"
 
 DISTRO_FEATURES_append = " wayland"
 DISTRO_FEATURES_remove = "x11"
+
+PREFERRED_PROVIDER_virtual/egl_hikey = "mali450-userland"
```

Comment:
why does that need to be here ? it would be more appropriate to be in the machine .conf file. any specific reason?